sdk/python/lib/pulumi/runtime/settings.py

```diff
@@ -195,6 +204,13 @@ def _set_stack(v: Optional[str]):
     SETTINGS.stack = v
 
 
+def get_rpc_manager() -> RPCManager:
```

Comment:
We haven't been consistent about this, especially in the early days of the Python SDK, but should we consider naming this `_get_rpc_manager()` (underscore prefix) to make it more clear that this is an internal thing that we don't expect users to import and use directly?

Reply:
Yes that makes sense